src/SignalR/clients/java/signalr/core/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -223,7 +223,11 @@ Transport getTransport() {
                         List<InvocationHandler> handlers = this.handlers.get(invocationMessage.getTarget());
                         if (handlers != null) {
                             for (InvocationHandler handler : handlers) {
-                                handler.getAction().invoke(invocationMessage.getArguments());
+                                try {
+                                    handler.getAction().invoke(invocationMessage.getArguments());
+                                } catch (Exception e) {
+                                    logger.error("Invoking client side method '{}' failed: {}", invocationMessage.getTarget(), e.getMessage());
```

Comment:
Would using `e.toString()` (or just `e`) instead allow us to get the exception type and stack trace in the logs similar to C#? If so, I'd prefer that.

```suggestion
                                    logger.error("Invoking client side method '{}' failed: {}", invocationMessage.getTarget(), e);
```

Reply:
> Would using `e.toString()` (or just `e`) instead allow us to get the exception type and stack trace in the logs

Yes, changed it